content-repo/create-docs.sh

```diff
@@ -149,4 +149,8 @@ else
     ./gendocs.py -t "${TARGET_DIR}" -d "${CONTENT_GIT_DIR}"
 fi
 
+echo "Generating Demisto class docs..."
+mv demistomock.py demisto.py
+pydoc-markdown -p demisto > ../docs/integrations/demistoclass.md
```

Comment:
Need to add the file `docs/integrations/demistoclass.md` to `.gitignore` so it is not committed

Reply:
When not running in NETLIFY you should use `pipenv run` so it is possible to run this locally. See line 146.